### PR TITLE
Docs: update "Example projects" a little

### PR DESCRIPTION
**Pull request by humitos**
Convert the table into sections and bullet list. It's a lot more readable and easy to follow. I also added "Markup language" which is important.

This page still needs some work, but baby steps.

📖 https://docs--11404.org.readthedocs.build/en/11404/examples.html

---
:books: Documentation previews :books:

- User's documentation (`docs`): https://docs--11404.org.readthedocs.build/en/11404/

- Developer's documentation (`dev`): https://dev--11404.org.readthedocs.build/en/11404/

### CODE REVIEW

File: docs/user/examples.rst [hunk 1]
```diff
@@ -1,53 +1,64 @@
...
+Antora with asciidoctor-kroki extension configured for AsciiDoc and Diagram as Code.
+
+Markup language
+  AsciiDoc
+Rendered version
+  https://example-antora-basic.readthedocs.io/
+Repository
+  https://github.com/man-chi/example-antora-basic/
 
 Real-life examples
```
Comment (ericholscher) [inline docs/user/examples.rst:56]: I wonder about putting this above the example projects? I think this list is super useful to get a sense of our product, and I'd love to include some more .com users there. I actually used it as a starting point for my sales call.

**Review by ericholscher — APPROVED** [review]

File: docs/user/examples.rst (hunk 1 above)
Comment (humitos) [reply docs/user/examples.rst:56]: I'm not sure to follow what you mean with "above the example projects". Can you expand on that?

Comment (ericholscher) [reply docs/user/examples.rst:56]: Link out to the awesome RTD projects list closer to the top of the page. Not necessary, but I think many users will likely use that as inspiration vs. people who actually want a full example project to start a docs project from.

Comment (humitos) [reply docs/user/examples.rst:56]: Ahhhh, the comment was on the "Real Life examples". I missed that on the first pass. Ok. Makes sense to me

**Pull request merged**